src/app/analyze/analyze-overview/analyze-overview.component.html

```diff
@@ -1,151 +1,252 @@
+                  <li class="list-group-item">
+                    <div class="list-group-item-text">
+                      Work Items <a>#201</a> assigned to you by <a>@magranfie</a> under wireframe label
+                    </div>
+                  </li>
+                  <li class="list-group-item">
+                    <div class="list-group-item-text">
+                      Work Items <a>#201</a> assigned to you by <a>@magranfie</a> under wireframe label
+                    </div>
+                  </li>
+                  <li class="list-group-item">
+                    <div class="list-group-item-text">
+                      Work Items <a>#201</a> assigned to you by <a>@magranfie</a> under wireframe label
+                    </div>
+                  </li>
+                  <li class="list-group-item">
+                    <div class="list-group-item-text">
+                      Work Items <a>#201</a> assigned to you by <a>@magranfie</a> under wireframe label
+                    </div>
+                  </li>
+                  <li class="list-group-item">
+                    <div class="list-group-item-text">
+                      Work Items <a>#201</a> assigned to you by <a>@magranfie</a> under wireframe label
+                    </div>
+                  </li>
+                  <li class="list-group-item">
+                    <div class="list-group-item-text">
+                      Work Items <a>#201</a> assigned to you by <a>@magranfie</a> under wireframe label
+                    </div>
+                  </li>
+                </ul>
+              </div>
+              <div class="card-pf-footer">
+                <a class="margin-bottom-10">
+                  <i class="pficon pficon-add-circle-o"></i>
+                  <span>Create Codebase</span>
+                </a>
+              </div>
             </div>
+          </div>
         </div>
-        <div class="col-md-2">
-            <div class="panel panel-300 panel-default clickable" (click)="router.navigate(['/', space.relationalData.creator.attributes.username, space.attributes.name ,'run']);">
-
-                <div class="panel-heading">
-                    <h2 class="panel-title text-center">Pipelines Executing</h2>
-                </div>
-                <div class="panel-body">
-                    <p class="text-center number">0</p>
-                </div>
-
+      </aside>
+    </div>
+    <div class="flex-grow-4 " >
+      <aside>
+        <div class="card-pf">
+          <div class="flex-grow-1 flex-container in-column-direction">
+            <h2 class="card-pf-title card-pf-utilization-details">
+              <span>Pipeline</span>
+              <span class="badge">0</span>
+            </h2>
+            <div class="blank-slate-pf">
+              <div class="blank-slate-pf-icon">&nbsp;</div>
+              <h1>There is no pipeline execution</h1>
+              <p>You will get updated once you commit/build
```

Comment:
@essjaysee - please take a look at this as well. It does not read correctly to me (text taken from the visual design).